src/readquartetdata.jl

```diff
@@ -805,7 +805,7 @@ descData(d::DataCF, filename::AbstractString) = descData(d, filename,0.7)
 
 
 """
-    summarizedataCF(d::DataCF)
+    `summarizedataCF(d::DataCF)`
```

Comment:
same here: there should be 4 spaces in front of the function signature, and the backticks should be removed.